### PR TITLE
EASY-1108 ddm-emd mapping

### PR DESCRIPTION
**Pull request by jo-pol**
fixes EASY-1108

#### When applied it will
* describe maintenance of DDM-EMD documentation

#### Where should the reviewer @DANS-KNAW/easy start?
Not much to choose from.

#### How should this be manually tested?
Follow the added description. If the generated packaging document shows differences, these updates should be fixed or committed.

#### related pull requests on github
repo                       | PR
-------------------------- | -----------------
easy-                      | [PR#](PRlink)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,24 @@ easy-schema
 A collection of XML schema's used by EASY. These schema's are published at 
 [https://easy.dans.knaw.nl/schemas/](https://easy.dans.knaw.nl/schemas/)
 
+
+Maintain sword-v1-packaging
+---------------------------
+
+The Sword-v1 [service document] (ignore login) refers to the [packaging document]
```
Comment (rvanheest) [inline README.md]: Have you tested these links? I'm pretty sure the first one doesn't. At least, not for me when I click `view` and try them...

**Comment by rvanheest** [comment]
Btw, I saw the build is failing because the project doesn't contain a `.travis.yml` file. Maybe someone should copy paste it from another project???

File: README.md [hunk 2]
```diff
@@ -8,27 +8,31 @@ A collection of XML schema's used by EASY. These schema's are published at
 Maintain sword-v1-packaging
 ---------------------------
 
-The Sword-v1 [service document] (ignore login) refers to the [packaging document]
-located next to the ddm [examples]. The `<article id='mapping'>` in the packaging document
+The Sword-v1 [service document] \(ignore login) refers to the published [packaging document], its source is
```
Comment (rvanheest) [inline README.md]: that link works fine now! Nice work. Didn't know you could escape things in Markdown...

One issue with this link though (didn't realize this before): it leads to something that is behind an authentication.

1. is that correct?
2. Should we refer to this '*private*' link on this repository, given that it is a **public** repository?

Would like to hear @janvanmansum on this.

File: README.md [hunk 3]
```diff
@@ -8,27 +8,31 @@ A collection of XML schema's used by EASY. These schema's are published at
...
 --------------------------
 
 Purpose:
  
-* Test an updated ddm.xsd (or the schema's it uses) by depositing via sword-V1 with new examples.
+* Test an updated `ddm.xsd` (or the schema's it uses) by depositing via sword-V1 with new/updated examples.
 * Verify the ingested metadata.
+* Check wether the description tab of a dataset (both view mode as edit mode for archivists) contains less than an XML downloaded from the same tab. It would require an update of the [form definition] or even more legacy code to fix such omissions.
+
+[form definition]: https://github.com/jo-pol/easy-app/blob/9ec1c7f0fc496250f797269c874a990d1c21decb/lib/easy-business/src/main/java/nl/knaw/dans/easy/domain/form/form-descriptions/unspecified.xml#L678-L681
```
Comment (rvanheest) [inline README.md]: Don't refer to your own fork! Refer to the 'real' thing itself.

File: README.md [hunk 4]
```diff
@@ -8,27 +8,31 @@ A collection of XML schema's used by EASY. These schema's are published at
...
 ---------------------------
 
-The Sword-v1 [service document] (ignore login) refers to the [packaging document]
-located next to the ddm [examples]. The `<article id='mapping'>` in the packaging document
+The Sword-v1 [service document] \(ignore login) refers to the published [packaging document], its source is
+located next to the [examples] directory. The `<article id='mapping'>` in the packaging document
 contains the DDM-EMD mapping. Subsequent articles show help content of the web forms.
 
 The mentioned articles are copy pasted from test results of the legacy components `ddm` and `web-ui`,
 in that order. These results are located at `target/pageDumps/swordPackagingFragmentHelp.html`.
```
Comment (rvanheest) [inline README.md]: Isn't `target` one of those folders that always gets excluded by `.gitignore`? Isn't there another link to refer to this html page?

File: README.md [hunk 5]
```diff
@@ -8,27 +8,31 @@ A collection of XML schema's used by EASY. These schema's are published at
...
-Note that the help content is generated from the code base, which might be behind on the actual
-editable texts in production.
+Note that the help content is generated from [editable] texts in the code base,
+the actual versions in production are leading in case of differences.
 
 [service document]: http://easy.dans.knaw.nl/sword/servicedocument
 [packaging document]: https://easy.dans.knaw.nl/schemas/docs/sword-v1-packaging.html
 [examples]: https://github.com/DANS-KNAW/easy-schema/tree/master/src/main/assembly/dist/docs
+[editable]: https://github.com/jo-pol/easy-app/tree/master/front-end/easy-webui/src/main/assembly/dist/res/example/editable/help
```
Comment (rvanheest) [inline README.md]: same as with `form definition`

File: README.md (hunk 4 above)
Comment (jo-pol) [inline README.md]: Please also read the phrase before: ... from test results... If not ignored I could have linked.

**Comment by jo-pol** [comment]
Copy-pasting yml: Looking at the bottom of https://travis-ci.org/DANS-KNAW easy-update-metadata-dataset has never been built despite the presence of a travis file.

File: README.md [hunk 6]
```diff
@@ -1,27 +1,32 @@
...
-refers to the published [packaging document], its source is
-located next to the [examples] directory. The `<article id='mapping'>` in the packaging document
-contains the DDM-EMD mapping. Subsequent articles show help content of the web forms.
+The Sword-v1 interface provides a machine-to-machine alternative for the manual deposit procudure of [EASY].
+The Sword-v1 [service document] \(you can ignore the login, only put requests require authentication)
+refers to the published [packaging document], its source is located next to the source of the [examples]
+directory. The `<article id='mapping'>` in the packaging document contains the DDM-EMD mapping.
+Subsequent articles show help content of the web forms.
+
+_Maintnance prerequisites_: Access to the legacy code of EASY.
```
Comment (rvanheest) [inline README.md]: "Maintnance" --> spelling incorrect

**Pull request merged**